CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 # Changelog
 
+## 0.58.0      2024-08-27
+
+* Log Insights: Revamp log parsing mechanism
+  - The new mechanism is more performant and allows for arbitrary
+    log_line_prefix settings. The new parsing mechanism is the default, but you
+    can set `db_log_line_prefix = legacy` in the config file or
+    `LOG_LINE_PREFIX=legacy` in the environment to revert to the old mechanism.
+* Log Insights: Redact parameters from utility statements by default
+  - Statements like `CREATE USER u WITH PASSWORD 'passw0rd'` can leak sensitive
+    data into Log Insights, so they are now redacted by default. Note that these
+    statements are usually very fast, and are normally only logged with Postgres
+    configurations like `log_min_duration_statement = 0` that log every statement.
```

Comment:
I think that's somewhat confusing, since we actually skip over slow query logs if `log_min_duration_statement = 0` is set.

Maybe we can instead state that such statements are only likely to be logged in edge cases, like a lock wait problem relating to the statement?